tests/db_functions/math/test_log.py

```diff
@@ -0,0 +1,34 @@
+import math
+from decimal import Decimal
+
+from django.db.models.functions import Log
+from django.test import TestCase
+
+from ..models import DecimalModel, IntegerModel
+
+
+class LogTests(TestCase):
```

Comment:
Why does this differ so much to the other tests? What is `assertLogEqual()` doing that isn't required by `LnTests`? Is this a hang over from before `assertAlmostEqual()` was being used and it wasn't correctly updated?

Also missing `test_float()`.

Reply:
Thanks, I have corrected it.

Thinking back, I might (wrongly) thought that some database use LOG(BASE, NUM) and some use LOG(NUM, BASE). However, today when I double checked, I found that all database use the form LOG(BASE, NUM) and only python use LOG(NUM, BASE).